app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is a class variable that stores references to `Person` instances by their names. Ensure that names are unique to avoid overwriting entries.